src/test/countComments.test.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { getCommentsCount } from '../modules/count.js';
+
+describe('Testing comments counter', () => {
+  test('Getting count for comments', () => {
+    const comments = [
+      {
+        comment: 'This is nice',
+        username: 'abi',
+        creation_date: '2022-11-06',
+      },
+      {
+        comment: 'Lets get together',
+        username: 'dele',
+        creation_date: '2022-11-07',
+      },
+      {
+        comment: 'Lets get together',
+        username: 'Bob',
+        creation_date: '2022-11-19',
+      },
+    ];
+    // const luanches = LaunchUI.renderLaunches();
+    const commentCount = getCommentsCount(comments);
+    expect(commentCount).toBe(3);
+  });
+});
+
+describe('Testing comments counter', () => {
+  test('Getting comment count for rocket when no comment is found', () => {
+    const data = [];
+    // const luanches = LaunchUI.renderLaunches();
```

Comment:
- [ ] Kindly avoid leaving commented pieces of code in your codebase. It makes your codebase unclean and is not a good practice :+1: